fastlane/Fastfile

```diff
@@ -0,0 +1,35 @@
+fastlane_version '2.71.0'
+fastlane_require 'aws-sdk-s3'
+fastlane_require 'erb'
+fastlane_require 'json'
+fastlane_require 'pathname'
+
+lane :publish_test do |options|
+  api_key = ''
+  unless ENV['IOS_API_KEY_ID'].nil? || ENV['IOS_API_KEY_ID'].empty? ||
+    ENV['IOS_API_ISSUER_ID'].nil? || ENV['IOS_API_ISSUER_ID'].empty? ||
```

Comment:
Is it possible to create releases with those environment variables ? 

By changing steps of circle-ci pipeline with those environment variables, it is possible to create release.

Reply:
I'm not sure what you mean, this is a custom script for Fastlane (was provided by @enahum) that uses those variables to upload to the App Store.

What needs to be changed here?

Reply:
You can create releases all you want, but to publish on TestFlight and subsequently on the Appstore you'll need to set the API_KEYS to be able to login to appleconnect.

What I would say here is that probably the name of the variables should be something like `MACOS_API_KEY` or `APPLE_API_KEY` instead of `IOS...`